main/MiNiFiMain.cpp

```diff
@@ -142,94 +159,39 @@ int main(int argc, char **argv) {
 #ifdef WIN32
 	if (!SetConsoleCtrlHandler(consoleSignalHandler, TRUE)) {
 		logger->log_error("Cannot install signal handler");
-		std::cerr << "Cannot install signal handler" << std::endl;
-		return 1;
+		return -1;
 	}
 
 	if (signal(SIGINT, sigHandler) == SIG_ERR || signal(SIGTERM, sigHandler) == SIG_ERR ) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+		logger->log_error("Cannot install signal handler");
 		return -1;
 	}
 #ifdef SIGBREAK
 	if (signal(SIGBREAK, sigHandler) == SIG_ERR) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+		logger->log_error("Cannot install signal handler");
 		return -1;
 	}
 #endif
 #else
 	if (signal(SIGINT, sigHandler) == SIG_ERR || signal(SIGTERM, sigHandler) == SIG_ERR || signal(SIGPIPE, SIG_IGN) == SIG_ERR) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+    logger->log_error("Cannot install signal handler");
```

Comment:
This looks very strangely indented.

Reply:
That's what you get for editing in Notepad++ in Windows. Thanks.